tools/scenario-player/scenario_player/runner.py

```diff
@@ -5,18 +5,18 @@
 import yaml
 from eth_utils import to_checksum_address
 from requests import RequestException, Session
-from toolz import first
-from web3 import HTTPProvider, Web3
-
-from raiden.accounts import Account
-from raiden.network.rpc.client import JSONRPCClient
 from scenario_player.exceptions import NodesUnreachableError, ScenarioError, TokenRegistrationError
 from scenario_player.utils import (
     TimeOutHTTPAdapter,
     get_gas_price_strategy,
     get_or_deploy_token,
     wait_for_txs,
 )
+from toolz import first
```

Comment:
Looks like isort is confused again ;)

Reply:
That was before i fixed my configs... should be good now.